cmake/FindR.cmake

```diff
@@ -38,50 +38,74 @@ endif()
 set(TEMP_CMAKE_FIND_APPBUNDLE ${CMAKE_FIND_APPBUNDLE})
 set(CMAKE_FIND_APPBUNDLE "NEVER")
 find_program(R_COMMAND R DOC "R executable.")
+message(STATUS "R_COMMAND found at: ${R_COMMAND}")
 find_program(R_SCRIPT_COMMAND Rscript DOC "Rscript executable.")
+message(STATUS "R_SCRIPT_COMMAND found at: ${R_SCRIPT_COMMAND}")
 set(CMAKE_FIND_APPBUNDLE ${TEMP_CMAKE_FIND_APPBUNDLE})
 
 if(R_COMMAND)
   # temporarily append ".dll" to the cmake find_library suffixes
   set(OLD_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES})
   set(CMAKE_FIND_LIBRARY_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES} ".dll")
 
-  execute_process(WORKING_DIRECTORY .
-                  COMMAND ${R_COMMAND} RHOME
```

Comment:
I think we can get this `R_COMMAND` to work if we add `r-base` to the build dependencies and do a little bit of what `cross-r-base` does during activation. I'll look into it a bit more.

Reply:
Yes I think that should be possible !

Reply:
I looked into using the R binary from the build prefix to have a working `R_COMMAND`, but it is more trouble than it's worth. This current solution is simpler so I think we should keep it.